__tests__/App-test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
```

Comment:
should we use [react-testing library](https://www.npmjs.com/package/@testing-library/react-native) instead?